shell/platform/windows/text_input_plugin.cc

```diff
@@ -446,8 +446,18 @@ void TextInputPlugin::SendStateUpdateWithDelta(const TextInputModel& model,
 
 void TextInputPlugin::EnterPressed(TextInputModel* model) {
   if (input_type_ == kMultilineInputType) {
+    std::u16string text_before_change =
+        fml::Utf8ToUtf16(active_model_->GetText());
+    std::u16string text = std::u16string({u'\n'});
```

Comment:
Since you're only using this when `enable_delta_model` is true, I'd inline this into that block. I'd actually also just pass `u'\n'` as the last argument to the `TextEditingDelta` constructor instead of creating a local.

Reply:
If I inline this we would end up getting the text/and selection from the model after the newline has been inserted and we want the text/and selection before the newline has been inserted. What I can do is call `model->AddText` in both conditional blocks. What do you think?